<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ask</class>
 <widget class="QWidget" name="Ask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>501</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/6.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(50,50,50,200);
color: #efefef;
font: 25 10pt &quot;Microsoft YaHei&quot;;
border-radius:10px;
padding:2px 4px</string>
    </property>
    <property name="placeholderText">
     <string>name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(50,50,50,200);
color: #efefef;
font: 25 10pt &quot;Microsoft YaHei&quot;;
border-radius:10px;
padding:2px 4px</string>
    </property>
    <property name="placeholderText">
     <string>code</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(50,50,50,200);
color: #efefef;
font: 25 10pt &quot;Microsoft YaHei&quot;;
border-radius:10px;
padding:2px 4px</string>
    </property>
    <property name="placeholderText">
     <string>model</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(50,50,50,200);
color: #efefef;
font: 25 10pt &quot;Microsoft YaHei&quot;;
border-radius:10px;
padding:2px 4px</string>
    </property>
    <property name="placeholderText">
     <string>type</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(50,50,50,200);
color: #efefef;
font: 25 10pt &quot;Microsoft YaHei&quot;;
border-radius:10px;
padding:2px 4px</string>
    </property>
    <property name="placeholderText">
     <string>camp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>450</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(50,50,50,200);
color: #efefef;
font: 25 10pt &quot;Microsoft YaHei&quot;;
border-radius:10px;
padding:2px 4px</string>
    </property>
    <property name="text">
     <string>Commit</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(50,50,50,300);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(50,50,50,200);
font: 10pt &quot;BIZ UDPGothic&quot;;
color: #6f6f6f;
padding:2px 4px;

</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
